ide-eclipse.html

```diff
@@ -398,8 +404,7 @@ <h5>4. Run the project</h5>
 
 <pre class="alert"><code class="alert">
 run {
-    ...
-    if (platform == 'win') {
+    if (System.getProperty("os.name").startsWith("Windows")) {
```

Comment:
Detect windows with the osdetector plugin.

Reply:
Fixed